### ISSUE TITLE
NPM ERR! missing script: webpack

### ISSUE THREAD

**Issue by KayDyshay** [npm/npm#18549]
#### I'm opening this issue because: When typing the cmd 'npm run webpack', I get the error 'missing script: webpack'.

  - [ ] npm is crashing.
  - [ ] npm is producing an incorrect install.
  - [X] npm is doing something I don't understand.
  - [ ] Other (_see below for feature requests_):

#### What's going wrong?
I've made sure to type in the script "start: webpack" & even tried "start: webpack.js", yet to no avail

### supporting information:

 - `npm -v` prints: 5.4.2
 - `node -v` prints: v6.11.2
 - `npm config get registry` prints: https://registry.npmjs.org/
 - Windows, OS X/macOS, or Linux?: macOS Sierra v 10.12.6
 - Network issues:
   - Geographic location where npm was run:
   - [ ] I use a proxy to connect to the npm registry.
   - [ ] I use a proxy to connect to the web.
   - [ ] I use a proxy when downloading Git repos.
   - [ ] I access the npm registry via a VPN
   - [ ] I don't use a proxy, but have limited or unreliable internet access.
 - Container:
   - [ ] I develop using Vagrant on Windows.
   - [ ] I develop using Vagrant on OS X or Linux.
   - [ ] I develop / deploy using Docker.
   - [ ] I deploy to a PaaS (Triton, Heroku).

I receive this in my debugging log report. I've also reviewed the [npm-scripts doc](https://docs.npmjs.com/misc/scripts), but cannot understand how the document can help.

0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/bin/node', '/usr/local/bin/npm', 'run', 'webpack' ]
2 info using npm@5.4.2
3 info using node@v6.11.2
4 verbose stack Error: missing script: webpack
4 verbose stack     at run (/usr/local/lib/node_modules/npm/lib/run-script.js:151:19)
4 verbose stack     at /usr/local/lib/node_modules/npm/lib/run-script.js:61:5
4 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:115:5
4 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:402:5
4 verbose stack     at checkBinReferences_ (/usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:357:45)
4 verbose stack     at final (/usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:400:3)
4 verbose stack     at then (/usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:160:5)
4 verbose stack     at gypfile_ (/usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:195:29)
4 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/read-package-json/read-json.js:190:5
4 verbose stack     at f (/usr/local/lib/node_modules/npm/node_modules/once/once.js:25:25)
5 verbose cwd /Users/kaneshapatterson/Documents/FCC_Projects
6 verbose Darwin 16.7.0
7 verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "webpack"
8 verbose node v6.11.2
9 verbose npm  v5.4.2
10 error missing script: webpack
11 verbose exit [ 1, true ]

**Comment by kenany** [comment]
Hello! So, the way `npm run webpack` works is that it looks in your current `package.json` for a defined `"webpack"` [script](https://docs.npmjs.com/misc/scripts), and then runs that. The error message you're seeing is trying to tell you that you have not defined a `"webpack"` script in your `package.json`, hence `npm run webpack` has no idea what you want it to do.

To solve this issue, all you have to do is add a new script to your `package.json`. It might look something like this, for example:

``` json
{
  "name": "package-name",
  "version": "1.0.0",
  "description": "package description",
  "scripts": {
    "webpack": "webpack --progress",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "dependencies": {},
  "devDependencies": {}
}
```

Note how this `package.json` has a `"webpack"` script that runs `webpack --progress`. You'll definitely want to change the command to whatever is most appropriate for your project.

Let me know if you run into any further `npm` issues :)

**Comment by KayDyshay** [comment]
Kenan,

Thanks for the follow-up. I found that it was a simple issue. I tried
running my project from a separate folder from where my webpack config file
was located. Once located, I uninstalled the webpack and followed the steps
of the project to my avail. I appreciate your reply and look forward to the
future support of the community!

All The Best,

Kanesha Patterson

- Kanesha P.
Web Developer & UI Design
